package.json

```diff
@@ -16,7 +16,8 @@
     "test": "node --import tsx --test ./src/**/*.test.*",
     "test:watch": "node --import tsx --test --watch ./src/**/*.test.*",
     "prepare": "husky install",
-    "type-check": "npx tsc"
+    "type-check": "npx tsc",
+    "build:css": "npx postcss src/styles/dark.css src/styles/light.css -d public/theme"
```

Comment:
you should add this script to "build" & optionally to "dev" that when CI is starting, themes files will prepare in public folder